### PR TITLE
Stateless formula

### PR DESCRIPTION
**Pull request by giorgiosironi**
Does not provide a database, but is able to host and run the container image for digests.

Depends on:
- https://github.com/elifesciences/builder-base-formula/pull/159
- https://github.com/elifesciences/builder-base-formula/pull/158

Part of https://github.com/elifesciences/jira-import/issues/4292

### CODE REVIEW

File: salt/digests/config/srv-digests-uwsgi.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[uwsgi]
```
Comment (giorgiosironi) [inline salt/digests/config/srv-digests-uwsgi.ini]: this is outdated and needs to be in line with https://github.com/elifesciences/digests/blob/develop/config/uwsgi.ini

File: salt/digests/config/home-deployuser-digests-containers.env [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ENVIRONMENT_NAME={{ pillar.elife.env }}
+NEW_RELIC_MONITOR_MODE={{ 'true' if pillar.elife.newrelic.enabled else 'false' }}
+NEW_RELIC_APP_NAME={{ salt['elife.cfg']('project.stackname') }}
+NEW_RELIC_LICENSE_KEY={{ pillar.elife.newrelic.license }}
+NEW_RELIC_LABELS=project:{{ salt['elife.cfg']('project.project_name', 'python-unknown-project') }};env:{{ pillar.elife.env }}
+AWS_DEFAULT_REGION={{ pillar.elife.aws.region }}
+AWS_ACCESS_KEY_ID={{ pillar.elife.aws.access_key_id }}
+AWS_SECRET_ACCESS_KEY={{ pillar.elife.aws.secret_access_key }}
+PYTHONOPTIMIZE=1
```
Comment (lsh-0) [inline salt/digests/config/home-deployuser-digests-containers.env]: if this is related to `python -O ...`, then I wonder if we should run the optimised code through the tests. Obviously `assert`s will be stripped out.

https://docs.python.org/2.7/tutorial/modules.html#compiled-python-files

File: salt/digests/config/srv-digests-uwsgi.ini [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+chdir=/srv/digests/
+uid=www-data
+gid=www-data
+virtualenv=/srv/digests/venv/
+wsgi-file = manage.py
```
Comment (lsh-0) [inline salt/digests/config/srv-digests-uwsgi.ini]: > this is outdated and needs to be in line with 
yes, this line looks a bit weird to me

File: salt/digests/config/home-deployuser-digests-containers.env (hunk 2 above)
Comment (giorgiosironi) [reply salt/digests/config/home-deployuser-digests-containers.env]: Will verify for this project, but `pytest` should take care of avoiding that https://docs.pytest.org/en/reorganize-docs/faq.html#what-s-this-magic-with-pytest-historic-notes

**Comment by giorgiosironi** [comment]
Have to merge to get this build off the ground, as the master-server configuration requires files to be in place. Should have committed a base folder structure first.

File: salt/digests/config/home-deployuser-digests-containers.env (hunk 2 above)
Comment (lsh-0) [reply salt/digests/config/home-deployuser-digests-containers.env]: ha - I didn't know pytest had been around so long. Django also had a 'magic removal' branch about the same time as pytest's own. Rewriting `assert` statements makes sense for a test runner.

**Comment by lsh-0** [comment]
> Should have committed a base folder structure first.

`./bldr project.new`

**Pull request merged**